_config.yml

```
@@ -66,6 +66,10 @@ theme_settings:
     corporate_logo: /img/follow/eleven.svg
     welcome_to_the_jungle: https://www.welcometothejungle.co/fr/companies/eleven-labs
     welcome_to_the_jungle_logo: /img/follow/welcome_to_the_jungle.svg
+    link_facebook: https://www.facebook.com/11Labs
+    link_twitter: https://www.twitter.com/eleven_labs/
+    link_linkedin: https://fr.linkedin.com/company/eleven-labs
+    link_wttj: https://www.welcometothejungle.com/fr/companies/eleven-labs
```

dommage d'avoir le fr en dur et de pas pouvoir alterner en fonction de la locale du site

---

tu peux faire 
```suggestion
    link_wttj: https://www.welcometothejungle.com/companies/eleven-labs
```

et ca redirect automatiquement. pareil pour linkedin

```
    link_linkedin: https://linkedin.com/company/eleven-labs
```

---

Modifié